docs/writing-docs/tutorials/basics.md

```diff
@@ -275,9 +275,11 @@ Within a validation section, you may specify which validators you want to enable
 
 ### Validators
 
-Currently, only one validator exists: the `BlocksExistValidator`. This validator looks at [highlighted blocks](../snippets.md#highlight) in the answer key and confirms that, for each highlighted block, the user's code contains at least one block of the same type. It does *not* validate the parameters passed into the block.
+Currently, only one validator exists: the `BlocksExistValidator`. This validator looks at blocks tagged with `//@validate-exists` or `//@highlight` comments in the answer key and confirms that, for each tagged block, the user's code contains at least one block of the same type. It does *not* validate the parameters passed into the block. For more information on highlighting, see [highlighted blocks](../snippets.md#highlight).
```

Comment:
I feel like the "for more information on highlighting" bit is misplaced. It's a bit confusing to me since the paragraph seems to be talking about both those tags, but the resources only links to one. I'm not really sure of a better place for it, though...

Reply:
I agree. I wasn't sure where to place it either. I could just remove it :) I only kept it because we had the link earlier.

Reply:
I think you could remove it.

Reply:
Could add another section in `snippets.md` for `validate-exists` and then refer to them:

```suggestion
Currently, only one validator exists: the `BlocksExistValidator`. This validator looks at blocks tagged with `//@validate-exists` or `//@highlight` comments (see [highlight](/writing-docs/snippets.md#highlight) and [validate-exists](/writing-docs/snippets#validate-exists) in code blocks) in the answer key and confirms that, for each tagged block, the user's code contains at least one block of the same type. It does *not* validate the parameters passed into the block.
```

Reply:
Or, maybe:

```suggestion
Currently, only one validator exists, the `BlocksExistValidator`, for [highlight](/writing-docs/snippets.md#highlight) and [validate-exists](/writing-docs/snippets#validate-exists) blocks. This validator looks at blocks tagged with `//@validate-exists` or `//@highlight` comments in the answer key and confirms that, for each tagged block, the user's code contains at least one block of the same type. It does *not* validate the parameters passed into the block.
```

Reply:
Good idea. Thanks for making that change!